### PR TITLE
Commits as reviews

### PR DESCRIPTION
**Pull request by snogge**

### CODE REVIEW

**Comment by tarsius** [comment]
This file (currently still) uses tabs for indentation, and it enforces that (`indent-tabs-mode`). However, it does not enforce `tab-width = 8` and you seem to have set it to 4.

IMO we should switch to using spaces. I've done that, rebased your commits on top, and pushed here again. (Also addressed two other whitespace nitpicks along the way.)

**Comment by tarsius** [comment]
Well, I tried to push, but you have not given me permissions to push to your repository. I have instead pushed to a branch by the same name in this repository.

You are aware that you have commit access to this repository, right? It would probably make sense if you pushed branches here instead.

I guess it is a good thing that you opened a pull-request, so we could address this minor issue, but generally you don't have to do that. You can of course open a pull-request, if you want my feedback on something in particular, but then please mention what that is in the pr description.

File: magit-gerrit.el [hunk 1]
```diff
@@ -141,6 +141,29 @@ Typical values would be \"publish\" or \"for\".")
...
+All query commands use the `--format=JSON', `--all-approvals',
+`--comments', and `--current-patch-set' options."
+  (with-temp-buffer
+	(apply #'magit-gerrit--ssh-cmd-insert "query"
+		   "--format=JSON"
+		   "--all-approvals" "--comments"
+		   "--current-patch-set"
+		   (concat "project:" project)
+		   query)
+	(message "%s" (buffer-string))
```
Comment (tarsius) [inline magit-gerrit.el:158]: Looks like a leftover debug statement.

Comment (snogge) [reply magit-gerrit.el]: Correct, I've removed it.

**Pull request merged**